src/domUpdates.js

```diff
@@ -231,14 +232,46 @@ const updateCurrentRecipe = recipeCard => {
   pageData.currentRecipeCard.outerRemoveBtn = recipeCard.closest('.individual-recipe-container').querySelector('.remove-panel');
 }
 
-const updateSaveButtons = (ID, addButton, removeButton) => {
-  if(checkSavedStatus(currentUser, ID)){
+const showGridFeedback = (recipeID, feedback) => {
+  const recipe = document.getElementById(recipeID);
+  const gridFeedback = recipe.querySelector('.grid-feedback');
+  gridFeedback.innerText = feedback;
+  gridFeedback.classList.add('show-feedback')
+  setTimeout(() => {gridFeedback.classList.remove('show-feedback')}, 751)
```

Comment:
this needs to be wrapped in a conditional if 
`if(recipe)` because this creates an error when recipe is undefined (after successful deletion in the your recipes view)

Reply:
done